scripts/prepare-web.sh

```
@@ -19,16 +19,26 @@ retry() {
-mv .vodozemac/dart/web/pkg/vodozemac_bindings_dart* ./assets/vodozemac/
-rm -rf .vodozemac
+# Generating the bindings costs ~4 minutes (clone + cargo install + wasm build)
+# and its output depends only on the pinned flutter_vodozemac version and this
+# script, so CI caches ./assets/vodozemac and skips the build when it restores.
+# See ci.instructions.md. Delete the files (or the cache) to force a rebuild.
+if ls ./assets/vodozemac/vodozemac_bindings_dart* >/dev/null 2>&1; then
+  echo "prepare-web: vodozemac bindings already present, skipping build"
+else
+  version=$(yq ".dependencies.flutter_vodozemac" < pubspec.yaml)
+  version=$(expr "$version" : '\^*\(.*\)')
+  git clone https://github.com/famedly/dart-vodozemac.git -b ${version} .vodozemac
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)